configs/src/deployments/arbitrum.json

```diff
@@ -0,0 +1,184 @@
+      "blockNumber": 145462363,
+      "deploymentTX": "0x4c355697e6e72ce882751c3c23b5aea768b39530fca9f3d48e5539ab904dc07a"
+    },
+    "PlaceholderSetup": {
+      "address": "0x6E924eA5864044D8642385683fFA5AD42FB687f2",
+      "blockNumber": 145462388,
+      "deploymentTX": "0x35fd51be4013bff16d0e87c2676b06cfe7f232fd22e5673a99a49e535d4c233c"
+    },
+    "PluginRepoBase": {
+      "address": "0x78744A9664D6A4dDbcd7B73c120535667E739ee3",
+      "blockNumber": 145462176,
+      "deploymentTX": "0x6459d6c62aac34ff322dc28e18432bf8a165d2cd3f1470e1bd162689e8492e27"
+    },
+    "PluginRepoFactory": {
+      "address": "0xE640Da5AD169630555A86D9b6b9C145B4961b1EB",
+      "blockNumber": 145462176,
+      "deploymentTX": "0x6459d6c62aac34ff322dc28e18432bf8a165d2cd3f1470e1bd162689e8492e27"
+    },
+    "PluginRepoRegistry": {
+      "address": "0xCe0B4124dea6105bfB85fB4461c4D39f360E9ef3",
+      "blockNumber": 145462169,
+      "deploymentTX": "0x1eb755729fa1102d5708652dbc520b68e66e536e92a2d8a6a669cbcc202289fd"
+    },
+    "PluginRepoRegistry_Implementation": {
+      "address": "0x95D563382BeD5AcB458759EE05b27DF2CB019Cc7",
+      "blockNumber": 145462163,
+      "deploymentTX": "0x900deff95319e3caf60cefb596b3c66b32cc08ee0dd60a86373232fdcca6e714"
+    },
+    "PluginSetupProcessor": {
+      "address": "0x308a1DC5020c4B5d992F5543a7236c465997fecB",
+      "blockNumber": 145462184,
+      "deploymentTX": "0x735155f214e08749f840b1b9df22ec8e3059f445501f0fc698757880f7f7829d"
+    },
+    "Plugin_ENSSubdomainRegistrar": {
+      "address": "0x6115b7F05cCeF2D883AB64c88ba7e4BD1b877215",
+      "blockNumber": 145462134,
+      "deploymentTX": "0x7efad88b9eb730f7470ee113b11d8f51033a9f33b02f5aeba4e9c6f734082a93"
+    },
+    "Plugin_ENSSubdomainRegistrar_Implementation": {
+      "address": "0x74b3B3504B5d6D1c6247009c9b1e3D8cFF7bd445",
+      "blockNumber": 145462127,
+      "deploymentTX": "0xfa1266e592c95ba55d14d17577a2b62ca8455caa1299fbfb5a9c129bf6b3213c"
+    },
+    "PublicResolver": {
+      "address": "0xc4ddBd0472C2688F5d278dAFC02E66EE176C8DE3",
+      "blockNumber": 145462028,
+      "deploymentTX": "0x4d80c170830c65a5c5534f3bdb4ae5a02ab990dd2764b3045e716a8f2cba2a7d"
+    },
+    "TokenVotingSetup": {
+      "address": "0x96E54098317631641703404C06A5afAD89da7373",
+      "blockNumber": 145462328,
+      "deploymentTX": "0xd4f13878dda0aa4a9600bdf07f23124aa135a848cce9bb6e20feaaf06a585b50"
+    },
+    "TokenVotingSetupImplementation": {
+      "address": "0x4f40F01b2944359A0E507b509a868C8DbFAd215B",
+      "blockNumber": 145462328,
+      "deploymentTX": "0xd4f13878dda0aa4a9600bdf07f23124aa135a848cce9bb6e20feaaf06a585b50"
+    },
+    "address-list-voting-repo": {
+      "address": "0xf415FF95166EF5D365fFB3bc6d1701f9e9ed7Df7",
+      "blockNumber": 145462398,
+      "deploymentTX": "0x8f40d205049edbeb4318406bb0decaf26bbeb26176df93fafb69913eb09cae22"
+    },
+    "address-list-voting-repo_Implementation": {
```

Comment:
I think taht since we are building this from scratch we should go for a casing strategy, because in the props of this file we have, camel case, kebab case, snake case and pascal case 😂

Reply:
:rofl:  agreed. I wanted to make the transition for the SDK as easy as possible but if you're up to to I will streamline the naming

Reply:
I'm 200% up to it, please choose one case and use it 😂

Reply:
better?

Reply:
> we should go for a casing strategy

1000% this. Didn't we create a task for it?

Please let's do it now 🙏

Reply:
> better?

We went from 4 to 2 so yep thats better :)

Reply:
where is the second?